task-list/app/assets/stylesheets/application.css

```diff
@@ -18,52 +53,158 @@ h2 {
   padding: 0;
 }
 
-section {
-  background-color: white;
-  width: 400px;
-  height: 100vh;
-  margin: 0 auto;
-}
-
-ul {
-  list-style-type: none;
-  margin: 0 auto;
-  padding: 0;
-  width: 400px;
-  align-self: center;
+legend {
+  font-size: 22.5px;
+  font-weight: bold;
+  padding-top: 30px;
+  padding-bottom: 10px;
 }
 
 li {
-  width: 400px;
   font: 200 15px/1.5 Helvetica, Verdana, sans-serif;
+  text-align: left;
+  padding-left: 20px;
 }
 
 li:last-child {
   border: none;
 }
 
-a {
+li a {
   text-decoration: none;
   color: #000;
   width: 300px;
   text-align: center;
-  font-weight: 200;
 }
 
-a:hover {
-  background: #f6f6f6;
+li a:hover {
   text-decoration: underline;
-  font-weight: 200;
 }
 
-li a {
-  text-decoration: none;
-  color: #000;
-  width: 300px;
-  text-align: center;
+#pencil_shavings {
+  display: block;
+  max-height: 300px;
+  margin: 0 auto;
+  padding: 20px;
 }
 
-li a:hover {
-  background: #f6f6f6;
-  text-decoration: underline;
+section {
+  background-color: white;
+  width: 400px;
+  height: 100vh;
+  margin: 0 auto;
+}
+
+#submit_button_holder input {
+  float: none;
+  margin: 20px;
 }
+
+ul {
+  max-width: 350px;
+  list-style-type: none;
+  margin: 0 auto;
+  padding: 0;
+  width: 400px;
+  align-self: center;
+}
+
+
+.buttonHolder{
```

Comment:
CSS naming conventions state that we separate multiple word classes by an underscore rather than using camel case